devlab/provision/generate_config.sh

```diff
@@ -34,9 +34,7 @@ while read key value
 do
     value=($value)
     value=${value[1]}
-    if [[ -n ${value} ]]; then
-      sed -i "s|<${key}>|${value}|g" ${result_config}
-    fi
+    sed -i "s|<${key}>|${value}|g" ${result_config}
```

Comment:
@realsystem, When user keep a blank key - it means that there will be an empty value (`''`). When user does not specify key at all - it means that there will be default value.